### ISSUE TITLE
Polar TEP

### ISSUE THREAD

**Issue by hlynch** [iceberg-project/ICEBERG-administration#4]
Met the lead developer on this project at Polar 2018: http://p-tep.polarview.org/
In his words "we did all this already". I think we need to dig into this and figure out what they have done that we can re-use, and where we are dong something new. (I don't know enough about the inner workings to know exactly where this differs, except obvious when it comes to the type of imagery being used.) Also, they would like to collaborate, so this is a discussion we need to have.

**Comment by bspitzbart** [comment]
I agree with Matteo. There are not a lot of specifics to go on as far as their infrastructure that we can re-use.  But if they do have some standard analysis tools such as atmospheric corrections, pan-sharpening etc., we can use or build off those.  Their applications (use-cases) are geological, focusing on icebergs, ice shelves, sea ice and do not mention the machine learning and wildlife population studies we are doing.

**Comment by mmacferrin** [comment]
Atmospheric corrections, pan-sharpening, etc is great, but it isn't exactly what we are doing here. Honestly I haven't used the PolarTEP platform so am unsure to what degree they've "already done" this or not, but I think the tools we're putting in place aren't the same as theirs. There is no animal/colony-counting routines in their platform, they aren't doing anything like this ASIFT project, and others. I mean there are online computing platforms out there like Google Earth Engine already, for sure, and if that's all that was needed than neither PolarTEP nor IceBerg would be necessary at all. But that doesn't inherently mean there isn't a ton of new stuff we could help provide to the community.

Anyway, again I haven't looked at it closely yet, but that's just my initial hot-take $.02, for what it's worth.

- Mike
